sympy/physics/vector/frame.py

```diff
@@ -834,8 +835,8 @@ def orient_space_fixed(self, parent, angles, rotation_order):
 
         >>> B1.orient_axis(N, N.z, q1)
         >>> B2.orient_axis(B1, N.x, q2)
-        >>> B.orient_axis(B2, N.y, q3)
-        >>> B.dcm(N).simplify() # doctest: +SKIP
+        >>> A.orient_axis(B2, N.y, q3)
+        >>> A.dcm(N).simplify() # doctest: +SKIP
```

Comment:
Shouldn't the SKIP tag be removed before checking?

Reply:
Oh yes, my bad.